swagger-ui/swagger-ui.js

```diff
@@ -30693,7 +30705,7 @@ Emitter.prototype.hasListeners = function(event){
  * TODO: combatible error handling?
  */
 
-module.exports = function(arr, fn, initial){  
+module.exports = function(arr, fn, initial){
```

Comment:
if this only modifies spacing, I wouldn't include it as this isn't a file that we maintain.